src/wrapper/objects/global_ref.rs

```diff
@@ -57,3 +64,36 @@ impl Drop for GlobalRef {
         }
     }
 }
+
+/// A detached global JVM reference that can be sent across threads. To do
+/// anything useful with it, it must be `attach`ed first.
+///
+/// Warning: detached global ref will leak memory if dropped. Attach to a
+/// `JNIEnv` to prevent this.
```

Comment:
Hmm, this isn't great. Could we perhaps get a JavaVM handle in the detatch method and then use that to get a JNIEnv during Drop so that we can prevent leaks?